converter/src/DelphesEDM4HepConverter.cc

```diff
@@ -130,6 +131,17 @@ namespace k4SimDelphes {
     // Make sure the shared collections are present
     registerGlobalCollections();
 
+    //filling the event header
+    auto* eventBranch = delphesTree->GetBranch("Event");
+
+    if (eventBranch) {
+      auto* delphesEvents = *(TClonesArray**)eventBranch->GetAddress();
+      auto* delphesEvent  = static_cast<HepMCEvent*>(delphesEvents->At(0));
+      createEventHeader(delphesEvent);
+    } else {
+      std::cout << "Could not find Event branch in Delphes tree. Not filling the EventHeader." << std::endl;
```

Comment:
Since we only create the Event branch for the Pythia8 based readers here, I think we can remove this output and rather open an issue for keeping track of other readers, where we do not yet populate the EventHeader.